pages/osx/fileicon.md

```diff
@@ -0,0 +1,20 @@
+# fileicon
+
+> a macOS CLI for managing custom icons for files and folders, as a programmatic alternative to interactively using Finder.
```

Comment:
```suggestion
> A macOS CLI for managing custom icons for files and folders.
> A programmatic alternative to interactively using Finder.
```

Reply:
This description seems a little too manual pagey...
Can it be simplified?

Reply:
I think it could be succinctly put as
```suggestion
> macOS CLI to manage custom files/folder icons.
```
what do you guys think?

Reply:
I'd prefer `file and folder icons`, but otherwise sounds good.

Reply:
But linter doesn't accept if the description starts with a lowercase letter right?

Reply:
Yeah, you're right.  In the long run that should be fixed, but I'll just put "A" back the beginning.

Suggestion updated.